docs/docs/configuration/detectors.md

```diff
@@ -30,6 +30,18 @@ detectors:
     device: usb:1
 ```
 
+Multiple PCIE/M.2 Corals:
```

Comment:
Please add comment that this only works when "Protection Mode" is off.

Reply:
Protection mode? Not familiar with that.

Reply:
Supervisor addon has that option
![image](https://user-images.githubusercontent.com/24575521/116444771-14219900-a823-11eb-8549-0fdeef35607b.png)

Reply:
This maybe specific to hassio addon as my guess is only /dev/apex_0 is passed under protection mode: https://github.com/blakeblackshear/frigate-hass-addons/blob/4256ca2731277dd9b84b02b42b91053414f144e4/frigate/config.json#L26

Reply:
I don't have supervisor to test it or comment about the use of said feature.

Reply:
Looks like a feature needs to be added to that to allow the user to modify the mappings.